website/docs/r/cloud_project_workflow_backup.html.markdown

```diff
@@ -0,0 +1,44 @@
+---
+layout: "ovh"
+page_title: "OVH: ovh_cloud_project_workflow_backup"
+sidebar_current: "docs-ovh-resource-cloud-project-workflow-backup"
+description: |-
+  Manage a workflow that schedules backups of public cloud instance 
+---
+
+# ovh_cloud_project_user
+
+Manage a worflow that schedules backups of public cloud instance.
+Note that upon deletion, the workflow is deleted but any backups that have been created by this workflow are not. 
+
+## Example Usage
+
+```hcl
```

Comment:
Example well formatted:

```
resource "ovh_cloud_project_workflow_backup" "my_backup" {
  service_name        = "xxxxxxxxxxxxxxxxxxxxxxxxxxxxxx"
  region_name         = "GRA5"
  cron                = "50 4 * * *"
  instance_id         = "xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxx"
  max_execution_count = "0"
  name                = "Backup workflow for instance"
  rotation            = "7"
}
```

For the next time, you can use the following command:`terraform fmt my-file.tf` :-)

Reply:
Updated !